styles.css

```diff
@@ -183,15 +183,21 @@ input {
 }
 
 .warning {
-  height: 10px;
-  width: 10px;
+  height: 20px;
+  width: 20px;
+  margin-right: 10px;
 }
+/* ******************************************************* */
+.warning-line {
+  border-bottom: 1px solid #f9ef90;
+}
+
 
```

Comment:
NOTE: We decided to color our error messages differently from the comp. The error message color specified by the prompt was indistinguishable from the meditation-theme color (#C278FD vs #EFB7EC). On an application where color-coding represents an important aspect of the UX, we thought it would be prudent to give the error messages coloring that was visibly different from the three categories. #F9EF90 is a shade of yellow that appears bright on the app.